README.md

```diff
@@ -13,6 +13,12 @@ and away you go with a full-stack blockchain and command line tool for transacti
 
 WARNING: Currently uses plain-text private keys for transactions and is otherwise not production ready.
 
+## Prerequisites
+
+* Go to https://golang.org/doc/install to install Golang. 
+* Once you have Golang installed you will also need to install the Golang package manager `glide`: https://github.com/Masterminds/glide. 
```

Comment:
`go get` succeeded but i wasn't able to run `make get_vendor_deps` without `glide`, i got this error:
```
make get_vendor_deps
go get github.com/Masterminds/glide
glide install
make: glide: No such file or directory
make: *** [get_vendor_deps] Error 1
```